connector/dialect.tdd

```diff
@@ -770,63 +413,88 @@
       <argument type='int' />
       <argument type='int' />
     </function>
+    <function group='operator' name='+' return-type='real'>
+      <formula>(%1 + %2)</formula>
+      <argument type='int' />
+      <argument type='real' />
+    </function>
+    <function group='operator' name='+' return-type='real'>
+      <formula>(%1 + %2)</formula>
+      <argument type='real' />
+      <argument type='int' />
+    </function>
     <function group='operator' name='+' return-type='str'>
       <formula>(%1 || %2)</formula>
       <argument type='str' />
       <argument type='str' />
     </function>
-    <function group='operator' name='+' return-type='datetime'>
-      <formula>(%1 + %2 * INTERVAL &apos;1 DAY&apos;)</formula>
-      <argument type='datetime' />
-      <argument type='real' />
-    </function>
-    <function group='operator' name='+' return-type='date'>
-      <formula>(%1 + %2 * INTERVAL &apos;1 DAY&apos;)</formula>
-      <argument type='date' />
-      <argument type='int' />
-    </function>
     <function group='operator' name='-' return-type='real'>
       <formula>(-%1)</formula>
       <argument type='real' />
     </function>
+    <function group='operator' name='-' return-type='int'>
```

Comment:
Good catch. Do we have unary addition anywhere in this file? If not, can we add it?

Reply:
Sure, added.